app/controllers/claims/support/support_users_controller.rb

```diff
@@ -31,6 +31,19 @@ def create
 
   def show; end
 
+  def remove; end
+
+  def destroy
+    authorize @support_user
+
+    if SupportUser::Remove.call(support_user: @support_user)
+      redirect_to claims_support_support_users_path, flash: { success: t(".success") }
+    else
+      flash[:alert] = t(".failure")
+      render :remove
```

Comment:
Tiny nitpick to inline the flash message.

```suggestion
      render :remove, alert: t(".failure")
```

Reply:
render doesn't work with flash, you have to declare the separately.